### DISCUSSION TITLE
Does changesets work with npm workspaces?

### DISCUSSION THREAD

**Discussion by runoncedev** [changesets/changesets#922] [Q&A] [answered]
I'm starting a new monorepo using Turborepo and they suggest using Changesets for versioning and publishing of packages in a registry.

But after going through [Changesets docs](https://github.com/changesets/changesets/blob/main/packages/cli/README.md#changesetscli-), only bolt and yarn workspaces are mentioned, so I wonder, is it possible to use Changesets with npm workspaces?

Thank you.

**Comment by Andarist** [comment, marked as answer]
It's possible - the only current drawback that I'm aware of is that you might want to run `npm install` after `changeset version` to update the lockfile with the information about the updated dependencies.
